src/pspm_init.m

```diff
@@ -411,6 +411,58 @@
     'import', @none, ...
     'data', 'wave');
 
+% documented
+defaults.chantypes(37) = ...
+    struct('type', 'gaze_x_lr_pp', ...

```

Comment:
@SerenXia Because some code seems to use one character to refer one side of the eyes, I do not recommend to use `lr` to denote "the combination of left and right eyes". I temporarily change it to `b` which denotes "bilateral". It would be easier to maintain some code, if you do not mind. We can change it back if it is preferred to use `lr` anyway. Thanks.

Reply:
> @SerenXia Because some code seems to use one character to refer one side of the eyes, I do not recommend to use `lr` to denote "the combination of left and right eyes". I temporarily change it to `b` which denotes "bilateral". It would be easier to maintain some code (e.g. line 173--188 in `pspm_find_valid_fixations_test`), if you do not mind. We can change it back if it is preferred to use `lr` anyway. Thanks.

Hi Teddy, again, this naming format was not determined by me. If you would like to change, you may need to discuss with Dominik.

Best regards,
Yanfang